### ISSUE TITLE
fingerprints

### ISSUE THREAD

**Issue by legion-s** [grip-unina/DMimageDetection#8]
Can you share the training code for fingerprints

**Comment by RCorvi** [comment]
Hi thank you for your interest. You can find the code to train the denoiser at the following [link.](https://github.com/cszn/DnCNN) As a remainder, in the paper we use a sigma equal to 1.

**Comment by legion-s** [comment]
Thanks for your answer, but I still want to know some training details:
1. The dataset used for training (dataset description) and the resolution of the training image (original or cropped)
2. When training, whether the convolutional layer of the neural network uses padding, and if so, what is the padding parameter?
3. Loss of use.
4. I see that the network you use when generating noise residuals does not use padding. Is padding not used during network training?
